src/cc/CCcontract.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <cstdint>
+#include "pubkey.h"
+#include "script/script.h"
+
+/// CC contract (Antara module) info structure that contains data used for signing and validation of cc contract transactions
+struct CCcontract_info
+{
+    /***
+     * Default ctor, sets everything to 0
+     */
+    CCcontract_info() { memset(this, 0, sizeof(CCcontract_info)); }
```

Comment:
Here the `memset` function is used to nullify fields of `CCcontract_info` class. Virtual table pointer (vptr) could be damaged, bcz `memset` doesn't know anything about vptr and `sizeof(CCcontract_info)` is virtual methods existence dependent. May be such default ctor with member initializer list will be more correct?
```
struct CCcontract_info
{

    uint8_t evalcode;
    uint8_t additionalTokensEvalcode2 = 0;
    char unspendableCCaddr[64];
    // < members skipped >
    CCcontract_info():   evalcode(0), additionalTokensEvalcode2(0),
                         unspendableCCaddr{0} /*, etc. */
                         {}
    virtual ~CCcontract_info() {}
    // < virtual methods skipped >
};
```

Reply:
yes i'd prefer to avoid using memset with c++ objects, as it may override members' constructor initialisations